complete/src/main/java/com/example/restservice/dao/Queue.java

```diff
@@ -1,18 +1,15 @@
 package com.example.restservice.dao;
 
 import java.util.List;
-import javax.persistence.CascadeType;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.OneToMany;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
Looks like IntelliJ automatically changed this.
Do we need project-wide editor settings?

Reply:
yeah, we will need to change that.